packages/fast-components-styles-msft/src/number-field/index.ts

```diff
@@ -18,6 +18,7 @@ const styles: ComponentStyles<NumberFieldClassNameContract, DesignSystem> = (
             height: density(defaultHeight)(designSystem),
             minHeight: toPx(minHeight),
             maxHeight: toPx(maxHeight),
+            padding: "6px 6px 6px 12px",
```

Comment:
LTR - https://developer.mozilla.org/en-US/docs/Web/CSS/padding this will modify the left value